app/src/main/java/com/keylesspalace/tusky/util/BlurHashDecoder.kt

```diff
@@ -102,10 +102,10 @@ object BlurHashDecoder {
                         b += color[2] * basis
                     }
                 }
-                bitmap.setPixel(x, y, Color.rgb(linearToSrgb(r), linearToSrgb(g), linearToSrgb(b)))
```

Comment:
huh, not only does `setPixel()` multiple checks but it's also implemented in native which means JNI. No wonder that array is faster